### PR TITLE
DevKnox pricing

### PR DESCRIPTION
**Pull request by dhilipsiva**

### CODE REVIEW

File: config/environment.js [hunk 1]
```diff
@@ -4,6 +4,7 @@ module.exports = function(environment) {
   var ENV = {
     isDevknox: false,
     isAppknox: false,
+    devknoxPrice: 9,  // This should also change in `mycroft/settings.py`
```
Comment (subho007) [inline config/environment.js]: @dhilipsiva why do we hardcode 9 here?

Comment (dhilipsiva) [inline config/environment.js]: @subho007 For time being. We will have a config endpoint. There is an Issue there.

Comment (subho007) [inline config/environment.js]: @dhilipsiva add a FIXME there. I'll approve the changes

Comment (dhilipsiva) [inline config/environment.js]: @subho007 Done!

**Pull request merged**